README.md

```diff
@@ -0,0 +1,148 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[contact\_form\] Contact Form](#-contact_form-contact-form)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [contact_form] <a name="about-project">Contact Form</a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[contact_form]** is a basic project to learn how the code review process will go.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<ul>
+  <li><a href="https://reactjs.org/">HTML</a></li>
+  <li><a href="https://expressjs.com/">CSS</a></li>
+  <li><a href="https://www.postgresql.org/">JS</a></li>
+</ul>
```

Comment:
Nice work sticking to the template here 👍 , but you need to edit the technologies with only the tech stack you used e.g ` HTML`, `CSS`, ` JavaScript`. Those included are not part of the project, and you would want to avoid giving your project views a negative impression and view of the stack used 🔍